sdk/nodejs/src/eventQueue.ts

```diff
@@ -38,15 +38,17 @@ export type EventQueueOptions = {
 
 export class EventQueue {
     private readonly logger: DVCLogger
+    private readonly reporter?: DVCReporter
     private readonly environmentKey: string
     eventFlushIntervalMS: number
     flushEventQueueSize: number
     maxEventQueueSize: number
     private flushInterval: NodeJS.Timer
     private flushInProgress = false
 
-    constructor(logger: DVCLogger, environmentKey: string, options: EventQueueOptions = {}) {
+    constructor(logger: DVCLogger, environmentKey: string, options: EventQueueOptions = {}, reporter?: DVCReporter) {
```

Comment:
couldn't this just be added as an options field rather than another constructor arg

Reply:
Also where is this actually set? I don't see where `DVCOptions.reporter` is passed into this reporter.

Reply:
Also DVCOptions is already being passed in here, so I think we should just grab it from there: 
<img width="1323" alt="image" src="https://user-images.githubusercontent.com/1219069/196220483-7e4b50ee-de83-454e-962a-69a808c934e5.png">

Reply:
Ah, missed this link in the chain. Added.